setup.py

```diff
@@ -41,8 +41,8 @@
     license='BSD',
     install_requires=[
         'oslo.config>=1.0,<=1.15',
-        'python-keystoneclient>=1.0,<=1.6',
```

Comment:
why remove those dependencies?

Reply:
sorry, did not check it when I created the PR, that was changed several months ago:)
for openstack Ussuri, it is using python3-keystoneclient-4.0.0, is it ok to remove just the upper constraint like <=1.6?